### PR TITLE
Add feedback for refactor

### PR DESCRIPTION
**Pull request by dpcairns**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -8,14 +10,14 @@ addButton.addEventListener('click', () => {
 
     console.log(input1.value);
     console.log(input2.value);
-    const firstValue = Number(input1.value);
-    const secondValue = Number(input2.value);
-    const sum = firstValue + secondValue;
+    const booger = Number(input1.value);
```
Comment (dpcairns) [inline app.js]: uhmm can we change these variable names?

File: mathUtils.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+// function declaration
+export function add(
+    /* DEFINING or DECLARING arguments goes here: generic names for input */ 
+    /* we dont care what the user calls them when they "pass them in" */
+    firstNumber, secondNumber) {
+
+    if (typeof firstNumber !== 'number' || typeof secondNumber !== 'number') {
```
Comment (dpcairns) [inline mathUtils.js:7]: is this check really necessary?

File: test/mathUtils.test.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    expect.equal(sum, expected);
+});
+
+
+test('it should take in 5 and -6 and return -1', function(expect) {
+    //Arrange
+    // Set up your arguments and expectations
+    const numberOne = 5;
+    const numberTwo = -6;
+    const expected = -1;
```
Comment (dpcairns) [inline test/mathUtils.test.js:27]: nice edge case test! 🏄
